apps/participants/views.py

```diff
@@ -197,18 +202,83 @@ def invite_participant_to_team(request, pk):
         }
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
-    serializer = InviteParticipantToTeamSerializer(
-        data=request.data,
-        context={"participant_team": participant_team, "request": request},
-    )
-
-    if serializer.is_valid():
-        serializer.save()
-        response_data = {
-            "message": "User has been successfully added to the team!"
-        }
+    '''
+    Sending unique email url to accept the invitation
+    '''
+    team_name = participant_team.team_name
+    encoded_team_id = urlsafe_base64_encode(force_bytes(pk)).decode()
+    encoded_email = urlsafe_base64_encode(force_bytes(email)).decode()
+    combined_url = "{}/{}".format(encoded_team_id, encoded_email)
+    web_url = request.data['url'].split("team")[0]
+    full_url = "{}invitation/{}".format(web_url, combined_url)
```

Comment:
See how hostname is being extracted from previously written functions. 
https://github.com/Cloud-CV/EvalAI/blob/a10d5275e9d0cdba0e4dcde7cfb5ae8def5ac82d/apps/challenges/views.py#L1735